core/src/banking_stage/consume_worker.rs

```diff
@@ -310,6 +316,9 @@ impl ConsumeWorkerMetrics {
         self.timing_metrics
             .find_and_send_votes_us
             .fetch_add(*find_and_send_votes_us, Ordering::Relaxed);
+        self.timing_metrics
+            .commit_count
```

Comment:
call this something else. it doesn't have a direct relationship with if we committed or not.

this is the number of times we have attempted to process transactions.

probably just call it `num_batches_processed`